### PR TITLE
Prebuilt docs

### PR DESCRIPTION
**Pull request by AniketSK**

### CODE REVIEW

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Prebuilt Quickstart
+nav: 5.3
+---
+
+100ms Prebuilt is a feature-rich video conferencing/live streaming [app](https://github.com/100mslive/AndroidPrebuiltDemo/) made using our prebuilt components. Prebuilt allows users to join sharable room links via desktop, meeting invites, coming soon as embeddable components in your apps.
```
Comment (PratimMallick) [inline docs/android/v2/quickstart/prebuilt-android.mdx:6]: we should specify that `(https://github.com/100mslive/AndroidPrebuiltDemo/)` is a demo of how to use 100ms prebuilt in android and not the prebuilt itself

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+[![](https://img.shields.io/badge/dynamic/xml.svg?label=100ms-Prebuilt&color=blue&query=%2F%2Fmetadata%2Fversioning%2Flatest&url=https%3A%2F%2Frepo1.maven.org%2Fmaven2%2Flive%2F100ms%2Froom-kit%2Fmaven-metadata.xml)](https://jitpack.io/#100mslive/room-kit)
+
+`implementation "live.100ms:room-kit:$version"`
+
+2. Launch Prebuilt from an activity/fragment.
+
+```kotlin
+val options = HMSPrebuiltOptions("My Name")
+HMSRoomKit.launchPrebuilt("tmz-qdi-cyp", this, options)
```
Comment (PratimMallick) [inline docs/android/v2/quickstart/prebuilt-android.mdx:32]: Specify that the first argument is `RoomCode`

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+1. Add the import for prebuilt.
+
+[![](https://img.shields.io/badge/dynamic/xml.svg?label=100ms-Prebuilt&color=blue&query=%2F%2Fmetadata%2Fversioning%2Flatest&url=https%3A%2F%2Frepo1.maven.org%2Fmaven2%2Flive%2F100ms%2Froom-kit%2Fmaven-metadata.xml)](https://jitpack.io/#100mslive/room-kit)
+
+`implementation "live.100ms:room-kit:$version"`
+
+2. Launch Prebuilt from an activity/fragment.
+
+```kotlin
+val options = HMSPrebuiltOptions("My Name")
```
Comment (PratimMallick) [inline docs/android/v2/quickstart/prebuilt-android.mdx:31]: We should specfy that `My Name` is the user Namw

File: docs/android/v2/quickstart/prebuilt-android.mdx (hunk 1 above)
Comment (AniketSK) [reply docs/android/v2/quickstart/prebuilt-android.mdx:6]: yes it says it's an app made with prebuilt

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 4]
```diff
@@ -0,0 +1,87 @@
+---
+title: Prebuilt Quickstart
+nav: 5.3
+---
+
+## Overview
+
+This guide will walk you through the steps to create a video conferencing app using Prebuilt components of the `room-kit` package.
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:8]:
```suggestion
This guide will walk you through the steps to create a video conferencing app using Prebuilt component of the `room-kit` package.
```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+Prebuilt is currently available for Android applications.
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:9]:
```suggestion

```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+```kotlin{1-2,12}
+import live.hms.roomkit.ui.HMSRoomKit
+import live.hms.roomkit.ui.HMSPrebuiltOptions
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val roomCode = "tmz-qdi-cyp"
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:50]:
```suggestion
        val roomCode = "<room-code>"
```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+```groovy
+dependencies { ....
+    implementation "live.100ms:room-kit:$GET_VERSION_FROM_ABOVE_BADGE"
+}
+```
+
+## Add the Prebuilt Component
+
+Look up how to get room codes in [this guide](/get-started/v2/get-started/prebuilt/room-codes/overview).
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:40]:
```suggestion
Look up how to get `room code` in [this guide](/get-started/v2/get-started/prebuilt/room-codes/overview).
```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+```kotlin{1-2,12}
+import live.hms.roomkit.ui.HMSRoomKit
+import live.hms.roomkit.ui.HMSPrebuiltOptions
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val roomCode = "tmz-qdi-cyp"
+        val userName = "My Name"
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:51]:
```suggestion
        val userName = "<my-name>"
```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+![Preview](/docs/v2/prebuilt_scr1.png)
+
+## Join a Room
+Here's what joining a room with another person looks like.
+
+![Join](/docs/v2/prebuilt_scr2.png)
+
+After joining with the app, join on web as well with the following link.
+
+[**Demo 100ms Pre-built link →**](https://examples-videoconferencing.app.100ms.live/meeting/tmz-qdi-cyp)
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:71]:
```suggestion

```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+You will see a preview of your own video and can turn it on or off.
+
+![Preview](/docs/v2/prebuilt_scr1.png)
+
+## Join a Room
+Here's what joining a room with another person looks like.
+
+![Join](/docs/v2/prebuilt_scr2.png)
+
+After joining with the app, join on web as well with the following link.
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:69]:
```suggestion

```

File: docs/android/v2/quickstart/prebuilt-android.mdx [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+[**Demo 100ms Pre-built link →**](https://examples-videoconferencing.app.100ms.live/meeting/tmz-qdi-cyp)
+
+## Additional Parameters
+Additional properties that can be specified are `userName` and `userId`.
```
Comment (saikatmitra91) [inline docs/android/v2/quickstart/prebuilt-android.mdx:74]: specified when ?

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/android/v2/quickstart/prebuilt-android.mdx (hunk 6 above)
Comment (AniketSK) [reply docs/android/v2/quickstart/prebuilt-android.mdx:49]: This is intended to be the demo link and people will be linked to an apk and the meeting link in the browser where they can try it out.

File: docs/android/v2/quickstart/prebuilt-android.mdx (hunk 7 above)
Comment (AniketSK) [reply docs/android/v2/quickstart/prebuilt-android.mdx:39]: In this context I believe it should be codes plural, but if you really think this is the right to refer to it in the doc I'll apply this.

Comment (saikatmitra91) [reply docs/android/v2/quickstart/prebuilt-android.mdx:39]: so if this this a quick start guide .. right now the developer / engineer is looking for a room code to instantiate the Prebuilt class. Not sure what will the engineer do with room codes ..

File: docs/android/v2/quickstart/prebuilt-android.mdx (hunk 6 above)
Comment (saikatmitra91) [reply docs/android/v2/quickstart/prebuilt-android.mdx:49]: lets not leak room codes. what if someone actually goes live with this room code .. 

not sure if the right customer will be billed in that case.

**Review by saikatmitra91 — APPROVED** [review]

**Pull request merged**